topics/go/algorithms/strings/permuation/permutation.go

```diff
@@ -0,0 +1,25 @@
+package strings
+
+import (
+	"sort"
+	"strings"
+)
+
+// IsPermutation check is two strings are permutations.
```

Comment:
Read the function description.  I think it is. `check if two strings` not is

Reply:
Fixed

Reply:
I don't see any of these changes when I view the file?

Reply:
That's funny it's not showing up. https://github.com/ardanlabs/gotraining/blob/permutation/topics/go/algorithms/strings/permuation/permutation.go

Reply:
check your commit/push

Reply:
delete the PR and re-create it.

Keep comment within 80 columns of width.